src/utils/urlOptions.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (function () {

```

Comment:
I created this as a small lib that's a bit more documented and tested https://github.com/ngokevin/urlgray/blob/master/index.js

If we don't want that, `queryParams` might be more accurate name than `urlOptions`